packages/api/tests/lambdas/sf-event-sqs-to-db-records/test-write-granules.js

```diff
@@ -363,6 +367,80 @@ test('writeGranules() saves granule records to Dynamo and RDS with same timestam
   t.is(pgRecord.updated_at.getTime(), dynamoRecord.updatedAt);
 });
 
+test('writeGranules() saves file records to RDS if RDS write is enabled and workflow status is "completed"', async (t) => {
```

Comment:
Nit:  We should probably rename these to 'Postgres'

Reply:
Nit: And probably should note that we're *also* validating Dynamo in the title

Reply:
Yup! Sure thing. I'm using `Postgres` here as opposed to `PostgreSQL` because it's less than formal and a little quicker to read than something user-facing.

On the second point... _are we_ validating Dynamo writes? We don't write files in Dynamo as a separate table or step in the process. They're written when a Granule is written. When we skip writing files here, we're only skipping Postgres writes. Does that make sense? I could be missing the point.